packages/core/src/lambda/vue/configEditor/samInvokeBackend.ts

```diff
@@ -348,17 +348,25 @@ export class SamInvokeWebview extends VueWebview {
      * Validate and execute the provided launch config.
      * TODO: Post validation failures back to webview?
      * @param config Config to invoke
+     * @param source Source of the invoke request
+     * @param useDebugger Whether to invoke with debugger attached
      */
-    public async invokeLaunchConfig(config: AwsSamDebuggerConfiguration, source?: string): Promise<void> {
+    public async invokeLaunchConfig(
+        config: AwsSamDebuggerConfiguration,
+        source?: string,
+        useDebugger: boolean = true
+    ): Promise<void> {
         const finalConfig = finalizeConfig(
             resolveWorkspaceFolderVariable(undefined, config),
             'Editor-Created Debug Config'
         )
         const targetUri = await this.getUriFromLaunchConfig(finalConfig)
         const folder = targetUri ? vscode.workspace.getWorkspaceFolder(targetUri) : undefined
 
-        // startDebugging on VS Code goes through the whole resolution chain
-        await vscode.debug.startDebugging(folder, finalConfig)
+        // Use the existing startDebugging method with the noDebug option
+        await vscode.debug.startDebugging(folder, finalConfig, {
+            noDebug: !useDebugger,
```

Comment:
? This is already supported by the existing (inherited from vscode) `noDebug` field: https://github.com/aws/aws-toolkit-vscode/blob/70ba83fc066cfe8a1f9f50e5eb73a5e1b51d42f4/packages/core/src/shared/sam/debugger/awsSamDebugger.ts#L179-L183